### PR TITLE
[WIP] Add a basic description of special ERB tags

### PR DESCRIPTION
**Pull request by ramya-n**
This is my very first pull request!

Should address the issue: #25, but is not quite ready to go yet. 
Feel free to review :)

Done during the [Codebar.io 24pullrequests special](http://codebar.io/events/24-pull-requests-2014) today.

With help from @fanaugen

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1152,6 +1152,36 @@ <h2 id="rails-commands">Important Rails commands</h2>
   </tr>  
 </table>
 
+<h2 id="rails-erb">ERB: Embedded Ruby</h2>
+<p>
+  In your views (that is, under <code>app/views</code> in your Rails app) you will find 
+  .html.erb files. <strong>ERB</strong> stands for <b>E</b>mbedded <b>Ruby</b>.
```
Comment (PragTob) [inline index.html]: Would be good to stick to `<strong>` here as opposed to `<b>` all the way through :)

File: index.html [hunk 2]
```diff
@@ -1152,6 +1152,36 @@ <h2 id="rails-commands">Important Rails commands</h2>
...
+  This just means that Rails processes some special tags in those files, and produces 
+  HTML output to send back to the user.
+</p>
+  There are only two ERB tags that you need to remember: <code><%= <i>ruby_code</i> %></code>,
+  and <code><% <i>ruby_code</i> %></code>
+<p>
```
Comment (PragTob) [inline index.html:1162]: the `<p>` tag should probably start above :)

**Comment by PragTob** [comment]
Thanks a lot for the work!!! =)

Looks really good, just some minor comments

**Comment by ramya-n** [comment]
Hi Tobias,

Thank you for letting me know about these, I just missed to update <p> tag, I shall change it to <strong>, I understand you want to keep it consistent :)
Kind Regards,
Ramya

**Comment by PragTob** [comment]
Hi, thanks a lot!

Only thing missing now is the description of `<% %>` - do you want to tackle that or shall I?

**Comment by ramya-n** [comment]
Hi Tobias,

Thank you, I have updated the changes,
Please let me know if you require any other changes,

Kind Regards,
Ramya

**Comment by PragTob** [comment]
Hi there, unfortunately that description isn't right - I'll fix it up. Thanks for the contribution!

**Pull request merged**